### ISSUE TITLE
no support to AveragePooling2D, BatchNormalization and Merge

### ISSUE THREAD

**Issue by samleoqh** [pplonski/keras2cpp#21]

**Comment by pplonski** [comment]
Dont be shy! You can add it :)

**Comment by samleoqh** [comment]
Hahaa, I will if I could manage to it -)
